app/Http/Controllers/Frontend/HotelController.php

```diff
@@ -31,13 +34,130 @@ public function index()
             'place_id',
             'hotels.created_at'
         ];
-
-        $hotels = Hotel::select($columns)
+        $query = Hotel::select($columns)
             ->with(['hotelServices' => function ($query) {
                 $query->join('services', 'hotel_services.service_id', '=', 'services.id');
-            }])
-            ->orderby('hotels.id', 'DESC')
+            }]);
+        $query = $this->placeCondition($request, $query);
+        $query = $this->orderCondition($request, $query);
+        $query = $this->checkinCondition($request, $query, $columns);
+        $hotels = $query->orderby('hotels.id', 'DESC')
             ->paginate(Hotel::ITEM_LIMIT);
-        return view('frontend.hotels.index', compact('hotels'));
+
+        /**
+         * Top 7 place booked most within the last month
+         */
+        $hintedPlaces = Place::topPlaces();
+        return view('frontend.hotels.index', compact('hotels', 'hintedPlaces'));
+    }
+
+    /**
+     * Get query after check place of hotels
+     *
+     * @param SearchHotelRequest $request request to display
+     * @param string             $query   query to change
+     *
+     * @return string
+     */
+    private function placeCondition($request, $query)
+    {
+        if ($request->has('hotelSourceArea')) {
+            /**
+             * Hotel belong to searched place
+             */
```

Comment:
Nếu là comment code cho method, biến thì dùng loại nhiều dòng như ri, ở đây là comment cho đoạn code nào đó - chỉ một dòng thì nên dùng `// Hotel belong to searched place`.
Chuyển hết comment trong method khác thành loại 1 dòng nha